src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -29,19 +30,16 @@ public class AnalysisService : IDisposable
 
         private const string PSSA_MODULE_NAME = "PSScriptAnalyzer";
 
+        private static readonly Version s_pssaMinimumVersion = new Version(1, 16);
```

Comment:
Considering 1.17 is shipping now, should we move to that version?

Reply:
Yes that crossed my mind. On one hand I feel like minimum version should encode what our lowest minimum supported version of PSSA is. On the other, we don't necessarily have a good answer for what that is, and I can't think of any downside to keeping it fresh...